test/performance/kueue/prerequisites/prometheus-sm.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Isn't this the same as https://github.com/kubernetes-sigs/kueue/blob/main/config/components/prometheus/monitor.yaml?

We have installation instructions here: https://github.com/kubernetes-sigs/kueue/blob/main/docs/setup/install.md#add-metrics-scraping-for-prometheus-operator

Reply:
I propose a slightly better approach: 

generating an effective Service monitor manifest for Kueue (based on the existing files in the repo) and injecting it into Prometheus manifests path of Clusterloader, so the Clusterloader can deploy it along with other Prometheus configurations when the tests starts, please have a look:
 
https://github.com/kubernetes-sigs/kueue/pull/462/commits/ca3bf8ff9fd4731ba17b1de87ed57b7bdf60ce63

Reply:
Okay, makes sense